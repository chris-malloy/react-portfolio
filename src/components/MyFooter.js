import React from 'react';
import { Icon,Footer } from 'react-materialize';

const MyFooter = () => {
  return (
    <Footer
      links={
        <ul>
          <li><a className="grey-text text-lighten-3" href="https://github.com/chris-malloy">Github<Icon tiny right>code</Icon></a></li>
          <li><a className="grey-text text-lighten-3" href="https://www.linkedin.com/in/christopher-malloy/">Linkedin<Icon tiny right>work</Icon></a></li>
          <li><a className="grey-text text-lighten-3" href="https://twitter.com/chris_malloy17">Twitter<Icon tiny right>group</Icon></a></li>
          <li><address><a href="mailto:chrismalloymusic@gmail.com">Email<Icon tiny right>mail</Icon></a></address></li>
        </ul>
      }
      id="contact"
      className='grey'
    >
      <h5>Thanks for visiting!</h5>
      <p>Want to ask me anything about coding?  I'd love to chat!</p>
    </Footer>
  )
}

export default MyFooter;