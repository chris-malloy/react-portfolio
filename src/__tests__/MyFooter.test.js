import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import MyFooter from '../components/MyFooter';

configure({ adapter: new Adapter() });

describe('MyFooter', () => {
  const myFooter = shallow(<MyFooter />);

  it('renders correctly', () => {
    expect(myFooter).toMatchSnapshot();
  });
});