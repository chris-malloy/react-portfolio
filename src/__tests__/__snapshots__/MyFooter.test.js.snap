// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyFooter renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MyFooter />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <h5>
          Thanks for visiting!
</h5>,
        <p>
          Want to ask me anything about coding?  I'd love to chat!
</p>,
      ],
      "className": "grey",
      "id": "contact",
      "links": <ul>
        <li>
                <a
                        className="grey-text text-lighten-3"
                        href="https://github.com/chris-malloy"
                >
                        Github
                        <Icon
                                right={true}
                                tiny={true}
                        >
                                code
                        </Icon>
                </a>
        </li>
        <li>
                <a
                        className="grey-text text-lighten-3"
                        href="https://www.linkedin.com/in/christopher-malloy/"
                >
                        Linkedin
                        <Icon
                                right={true}
                                tiny={true}
                        >
                                work
                        </Icon>
                </a>
        </li>
        <li>
                <a
                        className="grey-text text-lighten-3"
                        href="https://twitter.com/chris_malloy17"
                >
                        Twitter
                        <Icon
                                right={true}
                                tiny={true}
                        >
                                group
                        </Icon>
                </a>
        </li>
        <li>
                <address>
                        <a
                                href="mailto:chrismalloymusic@gmail.com"
                        >
                                Email
                                <Icon
                                        right={true}
                                        tiny={true}
                                >
                                        mail
                                </Icon>
                        </a>
                </address>
        </li>
</ul>,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Thanks for visiting!",
        },
        "ref": null,
        "rendered": "Thanks for visiting!",
        "type": "h5",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Want to ask me anything about coding?  I'd love to chat!",
        },
        "ref": null,
        "rendered": "Want to ask me anything about coding?  I'd love to chat!",
        "type": "p",
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <h5>
            Thanks for visiting!
</h5>,
          <p>
            Want to ask me anything about coding?  I'd love to chat!
</p>,
        ],
        "className": "grey",
        "id": "contact",
        "links": <ul>
          <li>
                    <a
                              className="grey-text text-lighten-3"
                              href="https://github.com/chris-malloy"
                    >
                              Github
                              <Icon
                                        right={true}
                                        tiny={true}
                              >
                                        code
                              </Icon>
                    </a>
          </li>
          <li>
                    <a
                              className="grey-text text-lighten-3"
                              href="https://www.linkedin.com/in/christopher-malloy/"
                    >
                              Linkedin
                              <Icon
                                        right={true}
                                        tiny={true}
                              >
                                        work
                              </Icon>
                    </a>
          </li>
          <li>
                    <a
                              className="grey-text text-lighten-3"
                              href="https://twitter.com/chris_malloy17"
                    >
                              Twitter
                              <Icon
                                        right={true}
                                        tiny={true}
                              >
                                        group
                              </Icon>
                    </a>
          </li>
          <li>
                    <address>
                              <a
                                        href="mailto:chrismalloymusic@gmail.com"
                              >
                                        Email
                                        <Icon
                                                  right={true}
                                                  tiny={true}
                                        >
                                                  mail
                                        </Icon>
                              </a>
                    </address>
          </li>
</ul>,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Thanks for visiting!",
          },
          "ref": null,
          "rendered": "Thanks for visiting!",
          "type": "h5",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Want to ask me anything about coding?  I'd love to chat!",
          },
          "ref": null,
          "rendered": "Want to ask me anything about coding?  I'd love to chat!",
          "type": "p",
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
